code/modules/admin/verbs/playsound.dm

```diff
@@ -45,9 +45,10 @@
 	if(!check_rights(R_SOUND))
 		return
 
+	var/vol = input("Select a volume for the sound", "Volume") as null|anything in list(100, 75, 50, 25, 5)
```

Comment:
```suggestion
	var/vol = input("Select a volume for the sound", "Play Local Sound", 50) as num
```
I think just making it a normal numeric input is best, since admins can set it to whatever they wish